data/json/items/generic.json

```diff
@@ -1503,6 +1505,7 @@
         "name": "science ID card",
         "description": "This ID card once belonged to a scientist of some sort.  It has a magnetic stripe on the back; perhaps it can be used on a control panel.",
         "price": 60000,
+        "price_postapoc": 5000,
```

Comment:
Because how easy it is to get the cards and how hard it can be to actually use them. Plus, you generally get a lot of them at once.
They would be worth a lot to factions, but not in a regular barter.
The military IDs can be "traded" into stuff in a bunker, but science ones would only be useful for those who would want to raid a lab for chemistry supplies and rare books. Those NPCs may later get special cased.